fabric-networking-api-v1/src/client/java/net/fabricmc/fabric/api/client/networking/v1/ClientPlayNetworking.java

```diff
@@ -41,6 +46,9 @@
  *
  * <p>This class should be only used on the physical client and for the logical client.
  *
+ * <p>See {@link ServerPlayNetworking} for information on how to use the new packet-object
+ * based API.
```

Comment:
```suggestion
 * <p>See {@link ServerPlayNetworking} for information on how to use the packet object-based API.
```
It won't be new in the future, and I think the hyphen needs to be on "...-based"